Resetting modules to system default. Reseting $MODULEPATH back to system default. All extra directories will be removed from $MODULEPATH.

Lmod is automatically replacing "xl/16.1.1-5" with "gcc/6.4.0".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/10.3.1.2-20200121

2020-05-14 16:04:04.925762: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:04:04.925762: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:04:04.925727: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:04:04.925740: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:04:04.925762: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:04:04.925727: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:04:11.610353 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:04:11.610352 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:04:11.610389 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:04:11.610572 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:04:11.610352 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:04:11.610541 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:04:11.610389 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:04:11.610571 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:04:11.610388 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:04:11.610541 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:04:11.610541 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:04:11.610572 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:04:11.809082 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:16: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:04:11.809082 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:16: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:04:11.809082 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:16: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:04:11.809286 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:16: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:04:11.809285 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:16: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:04:11.809390 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:19: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:04:11.809392 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:19: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:04:11.809393 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:19: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:04:11.809375 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:16: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:04:11.809577 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:19: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:04:11.809602 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:19: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:04:11.809577 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:19: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2020-05-14 16:04:11.827732: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:04:11.827725: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:04:11.827739: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:04:11.828366: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:04:11.828366: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:04:11.828366: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:04:11.829277: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x139b59a20 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:04:11.829311: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:04:11.829394: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x144fca8f0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:04:11.829414: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:04:11.829595: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x148a1a1c0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:04:11.829618: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:04:11.829865: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x16b7d9610 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:04:11.829886: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:04:11.829942: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x144009580 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:04:11.829966: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:04:11.829985: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x16e2fa020 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:04:11.830020: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:04:11.856986: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:04:11.856985: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:04:11.857436: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:04:11.857536: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:04:11.857538: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:04:11.857538: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:04:12.543953: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g06n03:147292] *** Process received signal ***
[g06n03:147292] Signal: Aborted (6)
[g06n03:147292] Signal code:  (-6)
[g06n03:147292] [ 0] [0x2000000504d8]
[g06n03:147292] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g06n03:147292] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g06n03:147292] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g06n03:147292] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g06n03:147292] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g06n03:147292] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g06n03:147292] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g06n03:147292] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g06n03:147292] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g06n03:147292] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g06n03:147292] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g06n03:147292] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g06n03:147292] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g06n03:147292] [14] python(_PyCFunction_FastCallDict+0xb0)[0x1316a5890]
[g06n03:147292] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x1316f2754]
[g06n03:147292] [16] python(+0x1c0a2c)[0x131790a2c]
[g06n03:147292] [17] python(_PyEval_EvalFrameDefault+0x604)[0x1317cafc4]
[g06n03:147292] [18] python(PyEval_EvalFrameEx+0x34)[0x131683244]
[g06n03:147292] [19] python(+0x1b4f08)[0x131784f08]
[g06n03:147292] [20] python(+0x1b6b74)[0x131786b74]
[g06n03:147292] [21] python(+0x1c0bac)[0x131790bac]
[g06n03:147292] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x1317cc0dc]
[g06n03:147292] [23] python(PyEval_EvalFrameEx+0x34)[0x131683244]
[g06n03:147292] [24] python(+0x1b4f08)[0x131784f08]
[g06n03:147292] [25] python(_PyFunction_FastCallDict+0x4dc)[0x1317877cc]
[g06n03:147292] [26] python(_PyObject_FastCallDict+0x2fc)[0x1316a5efc]
[g06n03:147292] [27] python(_PyObject_Call_Prepend+0x88)[0x1316aee48]
[g06n03:147292] [28] python(+0xdef74)[0x1316aef74]
[g06n03:147292] [29] python(PyObject_Call+0x74)[0x1316a5664]
[g06n03:147292] *** End of error message ***
2020-05-14 16:04:12.590538: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g06n03:147291] *** Process received signal ***
[g06n03:147291] Signal: Aborted (6)
[g06n03:147291] Signal code:  (-6)
[g06n03:147291] [ 0] [0x2000000504d8]
[g06n03:147291] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g06n03:147291] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g06n03:147291] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g06n03:147291] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g06n03:147291] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g06n03:147291] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g06n03:147291] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g06n03:147291] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g06n03:147291] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g06n03:147291] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g06n03:147291] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g06n03:147291] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g06n03:147291] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g06n03:147291] [14] python(_PyCFunction_FastCallDict+0xb0)[0x112485890]
[g06n03:147291] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x1124d2754]
[g06n03:147291] [16] python(+0x1c0a2c)[0x112570a2c]
[g06n03:147291] [17] python(_PyEval_EvalFrameDefault+0x604)[0x1125aafc4]
[g06n03:147291] [18] python(PyEval_EvalFrameEx+0x34)[0x112463244]
[g06n03:147291] [19] python(+0x1b4f08)[0x112564f08]
[g06n03:147291] [20] python(+0x1b6b74)[0x112566b74]
[g06n03:147291] [21] python(+0x1c0bac)[0x112570bac]
[g06n03:147291] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x1125ac0dc]
[g06n03:147291] [23] python(PyEval_EvalFrameEx+0x34)[0x112463244]
[g06n03:147291] [24] python(+0x1b4f08)[0x112564f08]
[g06n03:147291] [25] python(_PyFunction_FastCallDict+0x4dc)[0x1125677cc]
[g06n03:147291] [26] python(_PyObject_FastCallDict+0x2fc)[0x112485efc]
[g06n03:147291] [27] python(_PyObject_Call_Prepend+0x88)[0x11248ee48]
[g06n03:147291] [28] python(+0xdef74)[0x11248ef74]
[g06n03:147291] [29] python(PyObject_Call+0x74)[0x112485664]
[g06n03:147291] *** End of error message ***
2020-05-14 16:04:12.642680: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g06n03:147293] *** Process received signal ***
[g06n03:147293] Signal: Aborted (6)
[g06n03:147293] Signal code:  (-6)
[g06n03:147293] [ 0] [0x2000000504d8]
[g06n03:147293] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g06n03:147293] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g06n03:147293] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g06n03:147293] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g06n03:147293] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g06n03:147293] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g06n03:147293] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g06n03:147293] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g06n03:147293] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g06n03:147293] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g06n03:147293] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g06n03:147293] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g06n03:147293] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g06n03:147293] [14] python(_PyCFunction_FastCallDict+0xb0)[0x13d2b5890]
[g06n03:147293] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x13d302754]
[g06n03:147293] [16] python(+0x1c0a2c)[0x13d3a0a2c]
[g06n03:147293] [17] python(_PyEval_EvalFrameDefault+0x604)[0x13d3dafc4]
[g06n03:147293] [18] python(PyEval_EvalFrameEx+0x34)[0x13d293244]
[g06n03:147293] [19] python(+0x1b4f08)[0x13d394f08]
[g06n03:147293] [20] python(+0x1b6b74)[0x13d396b74]
[g06n03:147293] [21] python(+0x1c0bac)[0x13d3a0bac]
[g06n03:147293] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x13d3dc0dc]
[g06n03:147293] [23] python(PyEval_EvalFrameEx+0x34)[0x13d293244]
[g06n03:147293] [24] python(+0x1b4f08)[0x13d394f08]
[g06n03:147293] [25] python(_PyFunction_FastCallDict+0x4dc)[0x13d3977cc]
[g06n03:147293] [26] python(_PyObject_FastCallDict+0x2fc)[0x13d2b5efc]
[g06n03:147293] [27] python(_PyObject_Call_Prepend+0x88)[0x13d2bee48]
[g06n03:147293] [28] python(+0xdef74)[0x13d2bef74]
[g06n03:147293] [29] python(PyObject_Call+0x74)[0x13d2b5664]
[g06n03:147293] *** End of error message ***
2020-05-14 16:04:12.757777: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g06n03:147290] *** Process received signal ***
[g06n03:147290] Signal: Aborted (6)
[g06n03:147290] Signal code:  (-6)
[g06n03:147290] [ 0] [0x2000000504d8]
[g06n03:147290] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g06n03:147290] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g06n03:147290] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g06n03:147290] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g06n03:147290] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g06n03:147290] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g06n03:147290] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g06n03:147290] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g06n03:147290] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g06n03:147290] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g06n03:147290] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g06n03:147290] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g06n03:147290] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g06n03:147290] [14] python(_PyCFunction_FastCallDict+0xb0)[0x117985890]
[g06n03:147290] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x1179d2754]
[g06n03:147290] [16] python(+0x1c0a2c)[0x117a70a2c]
[g06n03:147290] [17] python(_PyEval_EvalFrameDefault+0x604)[0x117aaafc4]
[g06n03:147290] [18] python(PyEval_EvalFrameEx+0x34)[0x117963244]
[g06n03:147290] [19] python(+0x1b4f08)[0x117a64f08]
[g06n03:147290] [20] python(+0x1b6b74)[0x117a66b74]
[g06n03:147290] [21] python(+0x1c0bac)[0x117a70bac]
[g06n03:147290] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x117aac0dc]
[g06n03:147290] [23] python(PyEval_EvalFrameEx+0x34)[0x117963244]
[g06n03:147290] [24] python(+0x1b4f08)[0x117a64f08]
[g06n03:147290] [25] python(_PyFunction_FastCallDict+0x4dc)[0x117a677cc]
[g06n03:147290] [26] python(_PyObject_FastCallDict+0x2fc)[0x117985efc]
[g06n03:147290] [27] python(_PyObject_Call_Prepend+0x88)[0x11798ee48]
[g06n03:147290] [28] python(+0xdef74)[0x11798ef74]
[g06n03:147290] [29] python(PyObject_Call+0x74)[0x117985664]
[g06n03:147290] *** End of error message ***
2020-05-14 16:04:12.814735: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g06n03:147294] *** Process received signal ***
[g06n03:147294] Signal: Aborted (6)
[g06n03:147294] Signal code:  (-6)
[g06n03:147294] [ 0] [0x2000000504d8]
[g06n03:147294] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g06n03:147294] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g06n03:147294] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g06n03:147294] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g06n03:147294] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g06n03:147294] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g06n03:147294] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g06n03:147294] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g06n03:147294] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g06n03:147294] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g06n03:147294] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g06n03:147294] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g06n03:147294] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g06n03:147294] [14] python(_PyCFunction_FastCallDict+0xb0)[0x12b205890]
[g06n03:147294] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x12b252754]
[g06n03:147294] [16] python(+0x1c0a2c)[0x12b2f0a2c]
[g06n03:147294] [17] python(_PyEval_EvalFrameDefault+0x604)[0x12b32afc4]
[g06n03:147294] [18] python(PyEval_EvalFrameEx+0x34)[0x12b1e3244]
[g06n03:147294] [19] python(+0x1b4f08)[0x12b2e4f08]
[g06n03:147294] [20] python(+0x1b6b74)[0x12b2e6b74]
[g06n03:147294] [21] python(+0x1c0bac)[0x12b2f0bac]
[g06n03:147294] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x12b32c0dc]
[g06n03:147294] [23] python(PyEval_EvalFrameEx+0x34)[0x12b1e3244]
[g06n03:147294] [24] python(+0x1b4f08)[0x12b2e4f08]
[g06n03:147294] [25] python(_PyFunction_FastCallDict+0x4dc)[0x12b2e77cc]
[g06n03:147294] [26] python(_PyObject_FastCallDict+0x2fc)[0x12b205efc]
[g06n03:147294] [27] python(_PyObject_Call_Prepend+0x88)[0x12b20ee48]
[g06n03:147294] [28] python(+0xdef74)[0x12b20ef74]
[g06n03:147294] [29] python(PyObject_Call+0x74)[0x12b205664]
[g06n03:147294] *** End of error message ***
ERROR:  One or more process (first noticed rank 3) terminated with signal 6 (core dumped)

------------------------------------------------------------
Sender: LSF System <lsfadmin@batch3>
Subject: Job 96832: <nv> in cluster <summit> Exited

Job <nv> was submitted from host <login2> by user <yngtodd> in cluster <summit> at Thu May 14 16:03:15 2020
Job was executed on host(s) <1*batch3>, in queue <batch>, as user <yngtodd> in cluster <summit> at Thu May 14 16:03:45 2020
                            <42*g06n03>
</ccs/home/yngtodd> was used as the home directory.
</gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod> was used as the working directory.
Started at Thu May 14 16:03:45 2020
Terminated at Thu May 14 16:04:13 2020
Results reported at Thu May 14 16:04:13 2020

The output (if any) is above this job summary.

Resetting modules to system default. Reseting $MODULEPATH back to system default. All extra directories will be removed from $MODULEPATH.

Lmod is automatically replacing "xl/16.1.1-5" with "gcc/6.4.0".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/10.3.1.2-20200121

Error: nrs (1) should be evenly divided rs_per_host (6)
Error: Requested job constraints can never be met. Job was not scheduled. Check your request and try again.

------------------------------------------------------------
Sender: LSF System <lsfadmin@batch4>
Subject: Job 96869: <nv> in cluster <summit> Exited

Job <nv> was submitted from host <login2> by user <yngtodd> in cluster <summit> at Thu May 14 16:25:11 2020
Job was executed on host(s) <1*batch4>, in queue <batch>, as user <yngtodd> in cluster <summit> at Thu May 14 16:25:46 2020
                            <42*b22n14>
</ccs/home/yngtodd> was used as the home directory.
</gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod> was used as the working directory.
Started at Thu May 14 16:25:46 2020
Terminated at Thu May 14 16:26:05 2020
Results reported at Thu May 14 16:26:05 2020

The output (if any) is above this job summary.

Resetting modules to system default. Reseting $MODULEPATH back to system default. All extra directories will be removed from $MODULEPATH.

Lmod is automatically replacing "xl/16.1.1-5" with "gcc/6.4.0".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/10.3.1.2-20200121

2020-05-14 16:43:55.280451: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:55.280426: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:55.280426: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:55.280451: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:55.280426: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:55.280451: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:56.818907: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:56.818907: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:56.818918: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:56.818913: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:56.818919: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 16:43:56.818921: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:02.702456 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:02.702456 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:02.702554 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:02.702448 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:44:02.702638 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:02.702449 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:44:02.702640 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:02.702450 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:44:02.702638 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:02.702643 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:02.702643 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:02.702739 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:03.457844 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:03.457844 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:03.457843 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:03.457844 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:44:03.458034 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:03.457843 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 16:44:03.457844 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 16:44:03.458033 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:03.458033 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:03.458034 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:03.458034 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:03.458033 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 16:44:03.653538 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.653859 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.653830 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.653831 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.653934 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654113 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.653998 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654112 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654090 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654136 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654091 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654416 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654233 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654416 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654222 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654431 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654341 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654484 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654587 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654424 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654484 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 16:44:03.654650 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654703 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 16:44:03.654710 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2020-05-14 16:44:03.670908: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.671109: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.671332: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.671404: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.671469: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.671665: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.672568: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x13f139bb0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.672593: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.672802: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15020a150 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.672837: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.672872: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x13acb92c0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.672889: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.672957: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x14ff79cf0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.672971: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.673093: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x10d2d9ce0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.673116: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.673121: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x142669650 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.673138: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.700835: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:03.700839: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:03.700835: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:03.701011: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:03.701014: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:03.701015: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:03.991538: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.991614: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.991614: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.991727: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.991727: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.991788: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 16:44:03.993272: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x172fc9b10 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.993286: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.993465: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x118a19bb0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.993489: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.993474: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x12259a2e0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.993499: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.993684: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x16d769640 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.993705: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.993758: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15184a0d0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.993800: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:03.993800: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x14c8d9d50 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 16:44:03.993822: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 16:44:04.019675: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:04.019676: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:04.019674: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:04.020028: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:04.020026: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:04.020024: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 16:44:04.363351: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g23n14:107764] *** Process received signal ***
[g23n14:107764] Signal: Aborted (6)
[g23n14:107764] Signal code:  (-6)
[g23n14:107764] [ 0] [0x2000000504d8]
[g23n14:107764] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g23n14:107764] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g23n14:107764] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g23n14:107764] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g23n14:107764] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g23n14:107764] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g23n14:107764] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g23n14:107764] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g23n14:107764] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g23n14:107764] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g23n14:107764] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g23n14:107764] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g23n14:107764] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g23n14:107764] [14] python(_PyCFunction_FastCallDict+0xb0)[0x1256e5890]
[g23n14:107764] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x125732754]
[g23n14:107764] [16] python(+0x1c0a2c)[0x1257d0a2c]
[g23n14:107764] [17] python(_PyEval_EvalFrameDefault+0x604)[0x12580afc4]
[g23n14:107764] [18] python(PyEval_EvalFrameEx+0x34)[0x1256c3244]
[g23n14:107764] [19] python(+0x1b4f08)[0x1257c4f08]
[g23n14:107764] [20] python(+0x1b6b74)[0x1257c6b74]
[g23n14:107764] [21] python(+0x1c0bac)[0x1257d0bac]
[g23n14:107764] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x12580c0dc]
[g23n14:107764] [23] python(PyEval_EvalFrameEx+0x34)[0x1256c3244]
[g23n14:107764] [24] python(+0x1b4f08)[0x1257c4f08]
[g23n14:107764] [25] python(_PyFunction_FastCallDict+0x4dc)[0x1257c77cc]
[g23n14:107764] [26] python(_PyObject_FastCallDict+0x2fc)[0x1256e5efc]
[g23n14:107764] [27] python(_PyObject_Call_Prepend+0x88)[0x1256eee48]
[g23n14:107764] [28] python(+0xdef74)[0x1256eef74]
[g23n14:107764] [29] python(PyObject_Call+0x74)[0x1256e5664]
[g23n14:107764] *** End of error message ***
2020-05-14 16:44:04.413436: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g23n14:107766] *** Process received signal ***
[g23n14:107766] Signal: Aborted (6)
[g23n14:107766] Signal code:  (-6)
[g23n14:107766] [ 0] [0x2000000504d8]
[g23n14:107766] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g23n14:107766] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g23n14:107766] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g23n14:107766] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g23n14:107766] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g23n14:107766] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g23n14:107766] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g23n14:107766] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g23n14:107766] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g23n14:107766] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g23n14:107766] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g23n14:107766] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g23n14:107766] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g23n14:107766] [14] python(_PyCFunction_FastCallDict+0xb0)[0x12e1d5890]
[g23n14:107766] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x12e222754]
[g23n14:107766] [16] python(+0x1c0a2c)[0x12e2c0a2c]
[g23n14:107766] [17] python(_PyEval_EvalFrameDefault+0x604)[0x12e2fafc4]
[g23n14:107766] [18] python(PyEval_EvalFrameEx+0x34)[0x12e1b3244]
[g23n14:107766] [19] python(+0x1b4f08)[0x12e2b4f08]
[g23n14:107766] [20] python(+0x1b6b74)[0x12e2b6b74]
[g23n14:107766] [21] python(+0x1c0bac)[0x12e2c0bac]
[g23n14:107766] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x12e2fc0dc]
[g23n14:107766] [23] python(PyEval_EvalFrameEx+0x34)[0x12e1b3244]
[g23n14:107766] [24] python(+0x1b4f08)[0x12e2b4f08]
[g23n14:107766] [25] python(_PyFunction_FastCallDict+0x4dc)[0x12e2b77cc]
[g23n14:107766] [26] python(_PyObject_FastCallDict+0x2fc)[0x12e1d5efc]
[g23n14:107766] [27] python(_PyObject_Call_Prepend+0x88)[0x12e1dee48]
[g23n14:107766] [28] python(+0xdef74)[0x12e1def74]
[g23n14:107766] [29] python(PyObject_Call+0x74)[0x12e1d5664]
[g23n14:107766] *** End of error message ***
2020-05-14 16:44:04.460241: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g23n14:107762] *** Process received signal ***
[g23n14:107762] Signal: Aborted (6)
[g23n14:107762] Signal code:  (-6)
[g23n14:107762] [ 0] [0x2000000504d8]
[g23n14:107762] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g23n14:107762] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g23n14:107762] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g23n14:107762] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g23n14:107762] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g23n14:107762] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g23n14:107762] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g23n14:107762] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g23n14:107762] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g23n14:107762] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g23n14:107762] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g23n14:107762] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g23n14:107762] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g23n14:107762] [14] python(_PyCFunction_FastCallDict+0xb0)[0x107c85890]
[g23n14:107762] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x107cd2754]
[g23n14:107762] [16] python(+0x1c0a2c)[0x107d70a2c]
[g23n14:107762] [17] python(_PyEval_EvalFrameDefault+0x604)[0x107daafc4]
[g23n14:107762] [18] python(PyEval_EvalFrameEx+0x34)[0x107c63244]
[g23n14:107762] [19] python(+0x1b4f08)[0x107d64f08]
[g23n14:107762] [20] python(+0x1b6b74)[0x107d66b74]
[g23n14:107762] [21] python(+0x1c0bac)[0x107d70bac]
[g23n14:107762] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x107dac0dc]
[g23n14:107762] [23] python(PyEval_EvalFrameEx+0x34)[0x107c63244]
[g23n14:107762] [24] python(+0x1b4f08)[0x107d64f08]
[g23n14:107762] [25] python(_PyFunction_FastCallDict+0x4dc)[0x107d677cc]
[g23n14:107762] [26] python(_PyObject_FastCallDict+0x2fc)[0x107c85efc]
[g23n14:107762] [27] python(_PyObject_Call_Prepend+0x88)[0x107c8ee48]
[g23n14:107762] [28] python(+0xdef74)[0x107c8ef74]
[g23n14:107762] [29] python(PyObject_Call+0x74)[0x107c85664]
[g23n14:107762] *** End of error message ***
2020-05-14 16:44:04.518562: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g23n14:107765] *** Process received signal ***
[g23n14:107765] Signal: Aborted (6)
[g23n14:107765] Signal code:  (-6)
[g23n14:107765] [ 0] [0x2000000504d8]
[g23n14:107765] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g23n14:107765] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g23n14:107765] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g23n14:107765] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g23n14:107765] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g23n14:107765] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g23n14:107765] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g23n14:107765] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g23n14:107765] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g23n14:107765] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g23n14:107765] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g23n14:107765] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g23n14:107765] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g23n14:107765] [14] python(_PyCFunction_FastCallDict+0xb0)[0x12fc45890]
[g23n14:107765] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x12fc92754]
[g23n14:107765] [16] python(+0x1c0a2c)[0x12fd30a2c]
[g23n14:107765] [17] python(_PyEval_EvalFrameDefault+0x604)[0x12fd6afc4]
[g23n14:107765] [18] python(PyEval_EvalFrameEx+0x34)[0x12fc23244]
[g23n14:107765] [19] python(+0x1b4f08)[0x12fd24f08]
[g23n14:107765] [20] python(+0x1b6b74)[0x12fd26b74]
[g23n14:107765] [21] python(+0x1c0bac)[0x12fd30bac]
[g23n14:107765] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x12fd6c0dc]
[g23n14:107765] [23] python(PyEval_EvalFrameEx+0x34)[0x12fc23244]
[g23n14:107765] [24] python(+0x1b4f08)[0x12fd24f08]
[g23n14:107765] [25] python(_PyFunction_FastCallDict+0x4dc)[0x12fd277cc]
[g23n14:107765] [26] python(_PyObject_FastCallDict+0x2fc)[0x12fc45efc]
[g23n14:107765] [27] python(_PyObject_Call_Prepend+0x88)[0x12fc4ee48]
[g23n14:107765] [28] python(+0xdef74)[0x12fc4ef74]
[g23n14:107765] [29] python(PyObject_Call+0x74)[0x12fc45664]
[g23n14:107765] *** End of error message ***
2020-05-14 16:44:04.584359: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g23n14:107763] *** Process received signal ***
[g23n14:107763] Signal: Aborted (6)
[g23n14:107763] Signal code:  (-6)
[g23n14:107763] [ 0] [0x2000000504d8]
[g23n14:107763] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g23n14:107763] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g23n14:107763] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g23n14:107763] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g23n14:107763] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g23n14:107763] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g23n14:107763] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g23n14:107763] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g23n14:107763] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g23n14:107763] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g23n14:107763] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g23n14:107763] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g23n14:107763] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g23n14:107763] [14] python(_PyCFunction_FastCallDict+0xb0)[0x137015890]
[g23n14:107763] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x137062754]
[g23n14:107763] [16] python(+0x1c0a2c)[0x137100a2c]
[g23n14:107763] [17] python(_PyEval_EvalFrameDefault+0x604)[0x13713afc4]
[g23n14:107763] [18] python(PyEval_EvalFrameEx+0x34)[0x136ff3244]
[g23n14:107763] [19] python(+0x1b4f08)[0x1370f4f08]
[g23n14:107763] [20] python(+0x1b6b74)[0x1370f6b74]
[g23n14:107763] [21] python(+0x1c0bac)[0x137100bac]
[g23n14:107763] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x13713c0dc]
[g23n14:107763] [23] python(PyEval_EvalFrameEx+0x34)[0x136ff3244]
[g23n14:107763] [24] python(+0x1b4f08)[0x1370f4f08]
[g23n14:107763] [25] python(_PyFunction_FastCallDict+0x4dc)[0x1370f77cc]
[g23n14:107763] [26] python(_PyObject_FastCallDict+0x2fc)[0x137015efc]
[g23n14:107763] [27] python(_PyObject_Call_Prepend+0x88)[0x13701ee48]
[g23n14:107763] [28] python(+0xdef74)[0x13701ef74]
[g23n14:107763] [29] python(PyObject_Call+0x74)[0x137015664]
[g23n14:107763] *** End of error message ***
ERROR:  One or more process (first noticed rank 5) terminated with signal 6 (core dumped)

------------------------------------------------------------
Sender: LSF System <lsfadmin@batch3>
Subject: Job 96919: <nv> in cluster <summit> Exited

Job <nv> was submitted from host <login2> by user <yngtodd> in cluster <summit> at Thu May 14 16:42:58 2020
Job was executed on host(s) <1*batch3>, in queue <batch>, as user <yngtodd> in cluster <summit> at Thu May 14 16:43:32 2020
                            <42*g23n14>
                            <42*g23n15>
</ccs/home/yngtodd> was used as the home directory.
</gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod> was used as the working directory.
Started at Thu May 14 16:43:32 2020
Terminated at Thu May 14 16:44:05 2020
Results reported at Thu May 14 16:44:05 2020

The output (if any) is above this job summary.

